### DISCUSSION TITLE
Reverting StandardPixelRenderer

### DISCUSSION THREAD

**Discussion by ejeschke** [ejeschke/ginga#920] [Developers]
@pllim, something went awry in the merge of #845 which I didn't pick up on.  If you try to zoom using the number keys (or shifted number keys) it locks up the viewer.  You can zoom all you want with the mouse, or zoom to fit with the backquote key, or zoom in and out with the +/- keys, but you can't with the number keys.  Very strange.

What I did in commit d5ccc4435bbe2e2fbc0b2b9401fed1754cdcc9a2 is revert back to the previous renderer, and verified that it works properly.  I still want to replace that with the pipeline renderer, but it will take me a little time to figure out what is going on there.

**Comment by pllim** [comment]
Thanks for letting me know, @ejeschke !

**Comment by pllim** [comment]
@ejeschke , here is what GitHub Help Desk says, "When you create the discussion, you have that option or button available, but when someone else does they can see that option but you don't. The person who starts the conversation should be able to select the answered option."

Do you see the option to mark something as answered here? I cannot see it. If you don't, I have to let this Help Desk knows that their answer is wrong. Thanks!

**Comment by ejeschke** [comment]
> Do you see the option to mark something as answered here? 

Maybe that option was added after discussions debuted on github? IDK. I don't see the option for this thread.  I'll look for it on the next thread that is created.
